altair/utils/html.py

```diff
@@ -293,6 +326,12 @@ def spec_to_html(
         vlc = import_vl_convert()
         vl_version = vl_version_for_vl_convert()
         render_kwargs["vegaembed_script"] = vlc.javascript_bundle(vl_version=vl_version)
+    elif template == "olli":
+        olli_version = "2"
+        olli_adapters_version = "2"
```

Comment:
Should these be defined as global constants or exposed as parameters?

Reply:
Thought process was that as it's a very new feature, it's better to keep them internal for now. There are also only used in this one place. Regarding parameters, olli is currently being rewritten in Solid.js, it's API might change. Hence, version 3 might require changes to the rendering template -> Don't expect users to be able to just change a parameter to upgrade, we probably have to do it in Altair. Once that interface is more stable, having it as a renderer parameter sounds good!